### PR TITLE
adding xsede.expanse to the resource config

### PR DESCRIPTION
**Pull request by AymenFJA**
This PR adds `XSEDE.Expanse` to the resource config.

### CODE REVIEW

File: src/radical/pilot/configs/resource_xsede.json [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+        "resource_manager"            : "SLURM",
+        "lfs_path_per_node"           : "${LOCAL}",
+        "lfs_size_per_node"           : 1000000,
+        "cores_per_node"              : 128,
+        "gpus_per_node"               : 0,
+        "agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order" : ["FORK","MPIRUN"],
```
Comment (andre-merzky) [inline src/radical/pilot/configs/resource_xsede.json:28]: You probably want to remove `FORK` as that would not be able to place tasks on other nodes.

Comment (AymenFJA) [reply src/radical/pilot/configs/resource_xsede.json:28]: Great point, sorry missed that.

Comment (AymenFJA) [reply src/radical/pilot/configs/resource_xsede.json]: @andre-merzky  Done.

**Review by andre-merzky — APPROVED** [review]
LGTM

**Pull request merged**